jooq/src/test/scala/com/cerner/beadledom/jooq/DSLContextProviderImplSpec.scala

```diff
@@ -82,16 +85,21 @@ class DSLContextProviderImplSpec extends
           }
         }
 
+        val threads = mutable.ListBuffer[Thread]()
```

Comment:
Why was L88-L95 change required?

Reply:
Previously it started the threads and then as soon as it reached the closing `}` of the it-block it would assume the test passed even if not all the threads had finished. 

This new way waits until all 5 threads are finished or fails the test if it takes longer than 5 seconds.